includes/gateway/class-omise-payment-installment.php

```diff
@@ -72,12 +72,14 @@ public function payment_fields()
 		$currency   = get_woocommerce_currency();
 		$cart_total = WC()->cart->total;
 		$capabilities = $this->backend->capabilities();
+		$installmentMinLimit = $capabilities->getInstallmentMinLimits();
 
 		Omise_Util::render_view(
 			'templates/payment/form-installment.php',
 			array(
 				'installment_backends' => $this->backend->get_available_providers($currency, $cart_total),
-				'is_zero_interest'     => $capabilities ? $capabilities->is_zero_interest() : false
+				'is_zero_interest'     => $capabilities ? $capabilities->is_zero_interest() : false,
+				'installment_min_limit' => number_format($installmentMinLimit / 100) // subunit to base unit
```

Comment:
@aashishgurung why don't we use this function instead?

https://github.com/omise/omise-woocommerce/blob/master/includes/class-omise-money.php#L54

Reply:
I forgot it existed. I have updated the code and added `number_format` in the `convert_currency_unit` since it is suppose to be human readable. Thanks.